tools/tidb-binlog-monitor.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TiDB-Binlog Monitoring Metrics
+summary: This document introduces three levels of monitoring metrics of TiDB-Binlog.
```

Comment:
Please refer to other documents for the summary sentence pattern. Only several specific sentence patterns are used.